accounts-db/src/accounts_db.rs

```diff
@@ -2686,50 +2684,32 @@ impl AccountsDb {
     /// These should NOT be unref'd later from the accounts index.
     fn clean_accounts_older_than_root(
         &self,
-        candidates: &[RwLock<HashMap<Pubkey, CleaningInfo>>],
+        pubkey: &Pubkey,
```

Comment:
nit: this fn now works on only one account. Can we rename it to get rid of the plural?

Reply:
renamed.